### PR TITLE
Add CLI flag for acting on Sonar results

### PR DESCRIPTION
**Pull request by nahsra**
This change introduces the ability to act on Sonar input to the CLI specification.

### CODE REVIEW

File: cli.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ To guarantee a consistent user experience when using codemodder codemods, we off
 | --output          | the output file to produce (required) |
 | --output-format   | the format for the data output file (codetf or diff) |
 | --sarif           | comma-separated set of path(s) to SARIF file(s) to feed to the codemods |
+| --sonar-issues-json-file | the path to a file containing output from Sonar's Issues API |
```
Comment (drdavella) [inline cli.md:12]: I'm not going to pick a fight over it but I'd be happy with just `--sonar-json` for a slightly shorter name.

Comment (nahsra) [reply cli.md:12]: I guess I wanted to leave room for other Sonar things, or other Sonar formats, etc.

Comment (drdavella) [reply cli.md:12]: Maybe just `--sonar-issues-json` then? Since we don't have `--sarif-file`? Also we should add `--bikeshed` 😅

Comment (nahsra) [reply cli.md]: Done

**Review by drdavella — APPROVED** [review]

**Pull request merged**